server/methods.js

```diff
@@ -179,6 +180,29 @@ Meteor.methods({
     }
     return Instances.findOne({ _id: table_id }).slug;
   },
+  editadv(instanceid, newValues) {
+    if (_.isUndefined(this.userId)) {
```

Comment:
I know this works right now, but it won't in the future. That's because Meteor will eventually remove the implicit inclusion of `_` as a library. So you need to [import](https://guide.meteor.com/structure.html#intro-to-import-export) it.